git.md

```diff
@@ -261,10 +266,10 @@ Occasionally, a feature or component will mature to the extent that it should be
 
 Rather than copy-pasting code from the parent repository into a new one, it is preferable to retain the relevant commit history.
 
-So, the "split" should be done in such a way that the new repository contains only the desired files, and its commit history contains only the commits that have affected those files.  
+So, the "split" should be done in such a way that the new repository contains only the desired files, and its commit history contains only the commits that have affected those files.
```

Comment:
The "split" should be done in such a way that the new repository contains only the desired files, and its commit history contains only the commits that have affected those files.